### ISSUE TITLE
Radioulnar interosseous (IO) membrane tear

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10803]
**Preferred term label:**
Radioulnar interosseous (IO) membrane tear

**Synonyms**
Radioulnar IO membrane disruption; IO membrane transection; IO membrane injury

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Longitudinal instability of the forearm axis
Triangular fibrocartilage complex (TFCC) tear

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001361
- label: Radioulnar interosseous membrane tear
- parents: [Abnormal morphology of the radius (HP:0002818)](https://hpo.jax.org/app/browse/term/HP:0002818)
- pmid: PMID:27468369
- def: A tear in the fibrous sheet connecting the radius and ulna bones in the forearm.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10803
- synonyms:
